source/PreferencesPanel.cpp

```diff
@@ -364,41 +364,42 @@ void PreferencesPanel::DrawControls()
 	table.DrawAt(Point(-130, firstY));
 
 	static const string CATEGORIES[] = {
-		"Navigation",
-		"Weapons",
+		"Keyboard Navigation",
+		"Interface",
 		"Targeting",
-		"Navigation",
+		"Weapons",
 		"Interface",
-		"Fleet",
-		"Targeting"
+		"Fleet"
 	};
 	const string *category = CATEGORIES;
 	static const Command COMMANDS[] = {
 		Command::NONE,
 		Command::FORWARD,
 		Command::LEFT,
 		Command::RIGHT,
+		Command::AUTOSTEER,
 		Command::BACK,
 		Command::AFTERBURNER,
```

Comment:
While it is related to turning the ship, splitting back from forward, left, and right just seems weird.
```suggestion
		Command::BACK,
		Command::AUTOSTEER,
		Command::AFTERBURNER,
```
Arguably, it should also be below afterburner since it related to auto-piloting.
```suggestion
		Command::BACK,
		Command::AFTERBURNER,
		Command::AUTOSTEER,
```